History.md

```diff
@@ -0,0 +1,133 @@
+
+v3.0.0 / 2017-07-27
+===================
+
+  * Flush on first message (#113)
```

Comment:
I'm a bit confused here. What is the benefit of having the same release notes in changelog.md and gh releases? Especially when each line in the changelog generated by `git-changelog` links to a corresponding PR.